test/StringDiagram.lean

```diff
@@ -39,3 +39,27 @@ example {X Y : C} (f : X ⟶ Y) (g : 𝟙_ C ⊗ X ⟶ 𝟙_ C ⊗ Y) (w : False
     simp only [MonoidalCategory.id_whiskerLeft]
     -- now the widget works
     exact w.elim
+
+elab "normalize " t:term : term => do
```

Comment:
convention is to use
```suggestion
elab "normalize% " t:term : term => do
```
to make it clear that `normalize` is magic and not a regular function